lib/news.arc

```diff
@@ -390,22 +402,22 @@
 
 (defopr favicon.ico req favicon-url*)
 
-(def gen-css-url ()
-  (tag (link "rel" "stylesheet" "type" "text/css" "href" "news.css")))
-
 (mac npage (title . body)
-  `(tag html
-     (tag head
-       (gen-css-url)
-       (tag (link "rel" "shortcut icon" "href" favicon-url*))
-       (tag (meta "name" "viewport" "content" "width=device-width"))
-       (tag script (pr votejs*))
-       (tag title (pr ,title)))
-     (tag body
-       (center
-         (tag (table border 0 cellpadding 0 cellspacing 0 width "85%"
-                     bgcolor sand)
-           ,@body)))))
+  `(do 
+    (prn "<!DOCTYPE html>")
```

Comment:
Did you mean to enable the doctype? It's still disabling the stylesheet for me.

Reply:
Sorry, I thought you wanted it in, I'll comment it out and repush.

Apparently doctype declarations messing with CSS is a thing, and it may have to do with the formatting of the CSS, which I'll have to look into. Guess I know why HN doesn't have one.

Also since I just finished it, ol and li tags will work:

    (ul 
        (li (pr "foo"))
        (li (pr "foo"))
        (li (pr "foo")))

    (tag (ul "class" "list") 
          (li (pr "foo")) 
          (li (pr "bar"))
          (li (pr "baz")))

    lp just prints what's inside a list item
    (tag (ul "class" "list") 
          (lp "foo") 
          (lp "bar")
          (lp "baz"))

Reply:
I just suggested adding a note about why it was commented out. But we can fix that outside of this PR.